app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):

```

Comment:
`Elf` class does not implement all abstract methods from the `Player` class, inherit it from `ABC` to make abstract as well.